engine/consensus/compliance/engine.go

```diff
@@ -27,6 +27,9 @@ import (
 	"github.com/onflow/flow-go/utils/logging"
 )
 
+// defaultRangeResponseQueueCapacity maximum capacity of block range responses queue
+const defaultRangeResponseQueueCapacity = 100
```

Comment:
Yurii answered in [this thread](https://github.com/onflow/flow-go/pull/2935#discussion_r940704382):

> I think that updating compliance engine will render this change as not needed since there won't be hard dependency for validation between compliance and hotstuff. Because of it I would wait with changing collection cluster.

Reply:
👍💡  thanks for the reference